Documentation/gettingstarted/requirements_intro.rst

```diff
@@ -6,6 +6,7 @@ Make sure your Kubernetes environment is meeting the requirements:
 * Kubernetes >= 1.9
 * Linux kernel >= 4.9
 * Kubernetes in CNI mode
+* Key-Value store etcd >= 3.1.0 or consul >= 0.6.4
```

Comment:
While this improves the docs for the "External etcd" guide, this also now reports such a requirement for the "etcd operator" guide (http://docs.cilium.io/en/latest/gettingstarted/k8s-install-etcd-operator/) which actually requires neither `etcd` nor `consul` to be installed, as that guide will configure `etcd` as part of the guide.

Also, this text is only referred to from etcd guides so the Consul mention is not necessary. That option is already documented here: http://docs.cilium.io/en/latest/install/system_requirements/#summary

```
$ git grep requirements_intro Documentation/
Documentation/gettingstarted/k8s-install-etcd-operator.rst:.. include:: requirements_intro.rst
Documentation/gettingstarted/k8s-install-external-etcd.rst:.. include:: requirements_intro.rst
```